tests/parallel/t5000.nim

```diff
@@ -1,6 +1,6 @@
 discard """
   output: '''50005000'''
-  disabled: "true"
+    disabled: "true"
```

Comment:
Created around 2014-11 in a commit that references #1646
Disabled around 2014-12.
Working on my pc, but probably flaky
what is this even testing, though
Prob. delete

Reply:
Exterminate!

Seriously, I just can't be bothered with taking on fixing those libs right now.